platform.md

```diff
@@ -320,6 +325,7 @@ Usage:
 | `<layers>`        | `CNB_LAYERS_DIR`      | `/layers`                | Path to layers directory
 | `<image>`         |                       |                          | Tag reference to which the app image will be written
 | `<launch-cache>`  | `CNB_LAUNCH_CACHE_DIR`|                          | Path to a cache directory containing launch layers
+| `<layout>`        | `CNB_USE_OCI_LAYOUT`  | false                   | Enables the capability of resolving image from/to in OCI layout format on disk|
```

Comment:
```suggestion
| `<layout>`        | `CNB_USE_LAYOUT`  | false                   | Analyze image from OCI layout directory |
```

Reply:
Suggest `CNB_USE_OCI_LAYOUT` -> `CNB_USE_LAYOUT` to keep it terse and mirror `-daemon`